pages/signin.vue

```diff
@@ -0,0 +1,245 @@
+        firstName: {
+          required
+        },
+        lastName: {
+          required
+        },
+        email: {
+          required,
+          email,
+          maxLength: maxLength(99)
+        },
+        password: {
+          required
+        },
+        passwordConfirm: {
+          required,
+          sameAsPassword: sameAs('password')
+        }
+      }
+    },
+    methods: {
+      async signIn() {
+        const params = {
+          firstName: this.$v.form.firstName.$model,
+          lastName: this.$v.form.lastName.$model,
+          email: this.$v.form.email.$model,
+          password: this.$v.form.password.$model
+        };
+
+        try {
+          await this.$axios.post('/api/account/register', { params })
+            .then(response => {
+              this.user = response.data.user;
+
+              this.logIn();
+            });
+        } catch {
+          console.error('An error occurred.');
+        }
+      },
+      async logIn() {
+        await this.$auth.loginWith('local', {
+          data: {
+            "email": this.user.email,
+            "password": this.$v.form.password.$model
+          }
+        }).catch(e => {
+          console.error('Failed Logging In');
+        });
+
+          if (this.$auth.loggedIn) {
+            this.$router.push('/home');
+          }
+      }
+    }
+  }
+</script>
+
+<style scoped>
+    @import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800&display=swap');
+
+    #login {
+        font-family: 'Open Sans', sans-serif;
+        background-color: #f1c40f;
+        background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='28' height='49' viewBox='0 0 28 49'%3E%3Cg fill-rule='evenodd'%3E%3Cg id='hexagons' fill='%23ecf0f1' fill-opacity='0.4' fill-rule='nonzero'%3E%3Cpath d='M13.99 9.25l13 7.5v15l-13 7.5L1 31.75v-15l12.99-7.5zM3 17.9v12.7l10.99 6.34 11-6.35V17.9l-11-6.34L3 17.9zM0 15l12.98-7.5V0h-2v6.35L0 12.69v2.3zm0 18.5L12.98 41v8h-2v-6.85L0 35.81v-2.3zM15 0v7.5L27.99 15H28v-2.31h-.01L17 6.35V0h-2zm0 49v-8l12.99-7.5H28v2.31h-.01L17 42.15V49h-2z'/%3E%3C/g%3E%3C/g%3E%3C/svg%3E");
+        width: 100%;
+        height: 100vh;
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+        justify-content: center;
+    }
+    .login--window {
+        background-color: white;
+        height: 630px;
+        width: 350px;
+        margin: 0 auto;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        padding: 1em;
+        /* border: 1px solid black; */
+        border-radius: 7px;
+    }
+    h3 {
+        margin: 1em auto;
+        color: #1abc9c;
+        font-weight: 600;
+    }
+    input {
+        width: 100%;
+        height: 3em;
+        margin-top: 0.5em;
+        border-top: none;
+        border-right: none;
+        border-left: none;
+        font-size: 0.8em;
+    }
+    input:focus {
```

Comment:
There is a box-shadow when we focus the input, it's not in the file so it might be coming from somewhere else. I think we could remove it so the input looks clean and have a cleaner effect on focus.